javascript/react-js/state-and-props.md

```diff
@@ -232,6 +232,10 @@ For the state in functional components, it is a little bit different. Before the
 - How to handle state and props in a class component
 - Understanding setState()
 
+### Thinking in React
+
+An [article](https://reactjs.org/docs/thinking-in-react.html) by the React team on the thought process that should go into the developer's minds while building a react based apllication.
```

Comment:
Please capitalize the last "react" in the sentence.

Reply:
Done.